common/djangoapps/util/views.py

```diff
@@ -310,6 +313,24 @@ def _record_feedback_in_zendesk(
         u"\n".join(u"%s: %s" % (key, value) for (key, value) in additional_info.items() if value is not None)
     )
 
+    log.debug('Sending email to Support Team.')
```

Comment:
Please log some appropriate message? Some ticket information for which email is being generated.

Reply:
done